website/docs/docs-section/roadmap/v3/different-files.md

```diff
@@ -4,12 +4,12 @@ title: Different Types of Document
 sidebar_label: Different Types of Document
 ---
 
-OpenAttestation V3 is available for different types of document (Transferable Record & Verifiable Document) and different type of identity methods (`DNS-TXT`, `DNS-DID` & `DID`).
+OpenAttestation V3 is available for different types of document (including transferable record & verifiable document) and different type of identity methods (including `DNS-TXT`, `DNS-DID` & `DID`).
```

Comment:
not sure if it is correct use "including" here since both brackets contain exhaustive lists of what is available

Reply:
Removed "including"